allennlp/nn/util.py

```diff
@@ -339,6 +339,31 @@ def masked_mean(vector: torch.Tensor,
     return value_sum / value_count.clamp(min=eps)
 
 
+def masked_flip(padded_sequence: torch.Tensor,
+                sequence_lengths: List[int]) -> torch.Tensor:
+    """
+        To flip a padded tensor along the time dimension without affecting masked entries.
```

Comment:
Nit: Props for maintaining local consistency with the description of `masked_mean`, but "To flip" reads very strangely. "Flips" or "Flip" would be standard.